app/views/partials/leaderboard_table.html.erb

```diff
@@ -7,19 +7,22 @@
           <th>Rank</th>
           <th>Name</th>
           <th>Minutes</th>
+          <th>Hours</th>
         </tr>
       </thead>
       <% @leaderboards.each_with_index do |leaderboard,i| %>
         <tr>
           <td><%= i+1 %></td>
           <td><%= leaderboard.name %></td>
           <td><%= leaderboard.total_minutes %></td>
+          <td><%= leaderboard.total_minutes.to_f / 60 %></td>
```

Comment:
@mauro-oto I think rounding the numbers (like `.round(1)`) here could be useful, otherwise it could show up like this 
![screen shot 2016-04-20 at 12 29 25 pm](https://cloud.githubusercontent.com/assets/928745/14680404/989a9870-06f3-11e6-80ab-1da33ff372f8.png)